ncloud/resource_ncloud_block_storage.go

```diff
@@ -456,8 +486,9 @@ func deleteVpcBlockStorage(d *schema.ResourceData, config *ProviderConfig, id st
 	return nil
 }
 
-func detachBlockStorage(config *ProviderConfig, id string) error {
+func detachBlockStorage(d *schema.ResourceData, config *ProviderConfig, id string) error {
```

Comment:
```suggestion
func detachBlockStorage(config *ProviderConfig, id string) error {
```

`d *schema.ResourceData` 파라미터는 사용되지 않는것 같습니다.
호출쪽도 같이 제거하면 좋을것 같습니다~